jvb/src/main/resources/reference.conf

```diff
@@ -1,54 +1,54 @@
 videobridge {
   entity-expiration {
     # If an entity has no activity after this timeout, it is expired
-    timeout=1 minute
+    timeout =1 minute
```

Comment:
If you're adding whitespace this should have whitespace on both sides of the equal sign